eo-maven-plugin/src/main/java/org/eolang/maven/ShakeMojo.java

```diff
@@ -80,13 +78,12 @@ public final class ShakeMojo extends SafeMojo {
     void exec() {
         final long start = System.currentTimeMillis();
         final Collection<ForeignTojo> tojos = this.scopedTojos().withOptimized();
-        final Optimization optimization = this.optimization();
         final int total = new Threaded<>(
             new Filtered<>(
                 ForeignTojo::notShaken,
                 tojos
             ),
-            tojo -> this.shaken(tojo, optimization)
+            tojo -> this.shaken(tojo, this.optimization())
```

Comment:
@ArtemGet here, instead creating just one object you will create new object on every cycle iteration. So I believe this change is unnecessary

Reply:
Aggreed, reverted changes.